### PR TITLE
TravisCI: Add deployment stage

### PR DESCRIPTION
**Pull request by weidenka**
* Upload package to pypi for each tag
* The package version must be adjusted manually in setup.py
before uploading the tag

Pipenv:
* Add ipykernel
* Add shortcut to install jupyter kernel
`pipenv run installKernel`

### CODE REVIEW

File: setup.py [hunk 1]
```diff
@@ -18,11 +18,11 @@ def readme():
 
 
 setup(name='pycompwa',
-      version='0.0.1',
+      version='0.1.00',
```
Comment (spflueger) [inline setup.py]: is that supposed to be 0.1.00, not 0.1.0?

File: PyComPWA.cpp [hunk 2]
```diff
@@ -45,17 +45,39 @@ PYBIND11_DECLARE_HOLDER_TYPE(T, std::shared_ptr<T>);
...
+  
+  /// Reinitialize the logger with level INFO and disabled log file.
   ComPWA::Logging("INFO");
   py::class_<ComPWA::Logging, std::shared_ptr<ComPWA::Logging>>(m, "Logging")
       .def(py::init<std::string, std::string>(), "Initialize logging system",
-           py::arg("log_level"), py::arg("filename"))
-      .def(py::init<std::string>(), "Initialize logging system",
-           py::arg("log_level"));
+           py::arg("log_level"), py::arg("filename") = "")
+      .def_property("level", &ComPWA::Logging::getLogLevel,
```
Comment (spflueger) [inline PyComPWA.cpp]: What happens here if you set it to an illegal value?

File: .travis.yml [hunk 3]
```diff
@@ -1,83 +1,96 @@
...
+        user: __token__
+        password:
+          secure: bvlu0vG8oOYgLbNRkbZT5QO8AQDRlMuHPUn4U7PIECzQNjFC31Y8ugXTQOPdcNsACc8TR3z52uZ1JmuK4XI9++7HuzRwhMgKziCEwD/fBdZETxXq95LorbMaD8qOY43v5vJAeFDzouDoP3iBVTpJG3dZPk7OrHy0uSSGYnC0M/WqoFg/3Lfg5tlKz8doBbgGVG/vY/58GyDxu3GcSVg7c3A4CNYKk6FwXwbWo7itTzXMJFjWmlolFtNv9m4b4gLrPSvUf7A/BrovC9IYic2N+T/KnbVOBopQ9ggk+uf1eMv1emruNiqUaJ963DMPjE154hbR7Ykvz4ng/CsvI8enmiPg18RoS1DLGxe7LVr0RzZaGsW0gw8q6H6Noc0FzNXwLb5+fDWHJtz87rrVmZZz+ofVpo370Q9y7B6joJ5EBDxPwi/Ef9MBZMEMR1pArX9rnlkVpPit0nvsb+R4e5ZS5CHj7ArSxio5pXLMrEtJyJKpnaeCbjIp5b+UfsVwSbY6i3UpE0HSHDW/Nic02/d9PfgxuBkEp6jb30yajsfWroMpJQyKD/X1qIUO48deUO79Mnu33EGWnccYBbmhMxFNCQnfLVPwKAQm7YM1/WlUa6ictifxlLouG8wZjAUviBcvTjzDY53UUpS/PIkGPL9MrqjLJbsR0XwAwp4leprngyY=
+      # - provider: release # Create github release on each tagged commit
+      #   api_key:
+      #   file: "FILE TO UPLOAD"
+      #   skip_cleanup: true
+      #   draft: true
+      #   on:
+      #     tags: true
```
Comment (spflueger) [inline .travis.yml]: I would avoid uploading commented out code

File: PyComPWA.cpp (hunk 2 above)
Comment (weidenka) [reply PyComPWA.cpp]: Thats upto the C++ code

Comment (weidenka) [reply PyComPWA.cpp]: Basically nothing happens

File: .travis.yml (hunk 3 above)
Comment (weidenka) [reply .travis.yml]: Took me a while to find those lines so I would like to keep it

File: setup.py (hunk 1 above)
Comment (weidenka) [reply setup.py]: I guess thats the same version on pypi.

File: PyComPWA.cpp (hunk 2 above)
Comment (spflueger) [reply PyComPWA.cpp]: True

**Pull request merged**